modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -58,10 +66,54 @@ definition {
 		Click(locator1 = "TestrayManage#SAVE_BUTTON");
 	}
 
+	macro delete {
+		MouseOver(
+			key_value = "${valueName}",
+			locator1 = "TestrayManage#OPTIONS_NAME");
+
+		Click(locator1 = "TestrayDropdown#DROPDOWN_ITEM");
+
+		MenuItem.clickNoError(menuItem = "Delete");
+	}
+
+	macro editNewName {
+		TestrayManage.goToEditButton(valueName = "${valueName}");
+
+		Type(
+			locator1 = "TestrayManage#NEW_OPTION_NAME",
+			value1 = "${valueType}");
+
+		Click(locator1 = "TestrayManage#SAVE_BUTTON");
+	}
+
+	macro editOptions {
+		TestrayManage.goToEditButton(valueName = "${valueName}");
+
+		Type(
+			locator1 = "TestrayManage#NEW_OPTION_NAME",
+			value1 = "${valueType}");
+
+		Select(
+			locator1 = "TestrayManage#SELECT_CATEGORY",
+			value1 = "${Category}");
+
+		Click(locator1 = "TestrayManage#SAVE_BUTTON");
+	}
+
+	macro goToEditButton {
```

Comment:
@joelrocha68 the name of this macro is not exactly accurate. Maybe could it be "goToEditPage"?